Source/WebInspectorUI/UserInterface/Workers/Formatter/JSFormatter.js

```diff
@@ -463,6 +467,20 @@ JSFormatter = class JSFormatter
             return;
         }
 
+        if (nodeType === "ImportExpression") {
```

Comment:
Do you need to update the `import.js` test for these changes?

Reply:
This actually isn't necessary in my upcoming PR, which actually correctly handles ImportExpression from esprima all the way to our ast tree.